dockerfiles/pytorch-dev/install_rocm_from_tarballs.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+set -euo pipefail
+
+# Ensure wget is installed
+if ! command -v wget >/dev/null 2>&1; then
```

Comment:
Seems wget is not part of the Docker image by default but I guess curl might be? If so, you could switch to curl.

Reply:
Will try that

Reply:
Checked from installation and `curl `also is not default in the image - but I will go by `curl` -  updated the code